src/oscillator.js

```diff
@@ -442,7 +445,7 @@ class Oscillator {
       var now = p5sound.audiocontext.currentTime;
       this.stop(now);
       this.disconnect();
-      this.panner = null;
+      this.panner.dispose();
```

Comment:
I think this will throw an error if we're using the fallback non-`createStereoPanner` version. We could maybe add an empty `dispose` function to that version of the class so that this call just does nothing in that case?

Reply:
I think it will be better to add a `dispose` method to the fallback version.